proof_ipa.go

```diff
@@ -94,6 +94,7 @@ type SuffixStateDiffs []SuffixStateDiff
 type StemStateDiff struct {
 	Stem        [StemSize]byte   `json:"stem"`
 	SuffixDiffs SuffixStateDiffs `json:"suffixDiffs"`
+	Resurrected bool             `json:"resurrected"`
```

Comment:
This is going to be taking more space. I would do a `expiredStem` field like we have `poaStems`. But no need to take action on this right now, as long as it works it's fine. This is a later-stage optimization.